File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcResource/IfcResource.vb
Public Class IfcResource Inherits IfcObject
    Public Identification As IfcIdentifier
    Public LongDescription As IfcText

    ' === EXTENDED BY ===
    ' IfcConstructionResource

    ' === INVERSE CLAUSES ===
    ' ResourceOf : SET [0:?] OF IfcRelAssignsToResource FOR RelatingResource
End Class

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcResource/IfcResource.cs
public class IfcResource : IfcObject
{
    public IfcIdentifier Identification { get; set; }
    public IfcText LongDescription { get; set; }

    // === EXTENDED BY ===
    // IfcConstructionResource

    // === INVERSE CLAUSES ===
    // ResourceOf : SET [0:?] OF IfcRelAssignsToResource FOR RelatingResource
}
